config.js

```diff
@@ -1 +1,34 @@
-module.exports = {};
+module.exports = {
+  port: process.env.PORT || 8087,
+  db: {
+    database: process.env.DATABASE_NAME || 'asl',
+    host: process.env.DATABASE_HOST || 'localhost',
+    user: process.env.DATABASE_USERNAME || 'postgres'
+  },
+  emailer: process.env.EMAILER_SERVICE,
+  publicUrl: process.env.PUBLIC_UI,
```

Comment:
We don't have any situations where we send email to internal users yet do we?

Reply:
no not yet